Don't use MK_TESTS yet:  freebsd-buildutils >> 10.1~ will add it;
it defaults to "no" anyway.

--- a/src/sbin/devd/Makefile
+++ b/src/sbin/devd/Makefile
@@ -30,8 +30,4 @@
 
 CLEANFILES= y.output
 
-.if ${MK_TESTS} != "no"
-SUBDIR+=    tests
-.endif
-
 .include <bsd.prog.mk>
--- a/src/sbin/mdconfig/Makefile
+++ b/src/sbin/mdconfig/Makefile
@@ -8,8 +8,4 @@
 DPADD=	${LIBUTIL} ${LIBGEOM} ${LIBBSDXML} ${LIBSBUF} 
 LDADD=	-lgeom
 
-.if ${MK_TESTS} != "no"
-SUBDIR+=    tests
-.endif
-
 .include <bsd.prog.mk>
